src/translator/definitions.h

```diff
@@ -19,8 +19,8 @@ typedef AlignedVector<char> AlignedMemory;
 /// Memory bundle for all byte-arrays.
 /// Can be a set/subset of model, shortlist, vocabs and ssplitPrefixFile bytes.
 struct MemoryBundle {
-  AlignedMemory model{};      ///< Byte-array of model (aligned to 256)
-  AlignedMemory shortlist{};  ///< Byte-array of shortlist (aligned to 64)
+  std::vector<std::shared_ptr<AlignedMemory>> models{};  ///< Byte-array of model (aligned to 256)
```

Comment:
Was `std::vector<AlignedMemory>` not working?  It has move semantics.

Reply:
I went for the same lifetime semantics as is implemented for the vocabulary. 

I'll check if shared_ptr can be avoided

Reply:
Works without `shared_ptr`. The vocabulary use-case avoids a second allocation when using joint-vocabularies.

While a user could specify the same model twice in the ensemble, I don't think it's necessary provide a more memory efficient behaviour in that instance, because they really should be using `weights`  in their `config.yml` to achieve that anyway.